tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "compilerOptions": {
     "allowJs": true,
     "target": "esnext",
-    "moduleResolution": "node",
-    "module": "commonjs",
+    "moduleResolution": "node10",
+    "module": "node16",
```

Comment:
should we upgrade here?

Reply:
Assuming you mean upgrading to [node16/nodenext](https://www.typescriptlang.org/docs/handbook/modules/theory.html#the-module-output-format) module resolution, but that would probably involve changing a large number of imports with relatively few benefits, so worth holding off on that IMO